src/3D/Camera.cpp

```diff
@@ -21,6 +21,25 @@
 
 using namespace openblack;
 
+Camera::Camera(glm::vec3 position, glm::vec3 rotation)
+{
+	_position = position;
+	_rotation = glm::radians(rotation);
+	_dv = glm::vec3(0.0f, 0.0f, 0.0f);
+	_dwv = glm::vec3(0.0f, 0.0f, 0.0f);
+	_drv = glm::vec3(0.0f, 0.0f, 0.0f);
+	_viewCenterLand = _position;
+	_projectionMatrix = glm::mat4(1.0f);
+	_velocity = glm::vec3(0.0f, 0.0f, 0.0f);
+	_rotVelocity = glm::vec3(0.0f, 0.0f, 0.0f);
+	_groundHeight = 0.0f;
+	_groundHeightStart = 0.0f;
+	_accelFactor = 0.001f;
+	_movementSpeed = 4.0f;
+	_maxMovementSpeed = 0.005f;
+	_maxRotationSpeed = 0.005f;
+}
```

Comment:
The initializer list was better. Just that the order had to match the declaration